addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -1033,8 +1033,9 @@ const Many2OneAvatar = FieldMany2One.extend({
     _renderReadonly() {
         this.$el.empty();
         if (this.value) {
+            const fieldName = ['res.partner', 'res.users', 'hr.employee'].includes(this.field.relation) ? 'avatar_128' : 'image_128';
```

Comment:
This is not good to directly reference models in the framework, especially from `hr` which is not a dependency.
At worse at least make a new method to return the field name, so that you can override in hr to include employee.

Reply:
it's an "avatar" field but there are other models that use it, which don't have an avatar_128 field

Reply:
I agree that it's ugly like this. What's your suggestion on how to implement this?